src/sync/lookupApiUtils.js

```diff
@@ -256,8 +257,8 @@ export const getPatientHistoryResponseProcessor = ({
 
 export const processPatientResponse = response => {
   const result = processResponse(response);
-  return result.map(
-    ({
+  return result.map(patient => {
```

Comment:
I changed this because patients retrieved from mSupply for HL7 patients don't have all these fields, if we destruct straight away it was failing, but I see you are right. I Ican keep it desctructing as the in param and then use the check to properly only set if existing

Reply:
Latest changes on the server side branch mean that all the patients should have all the the usual `name` fields 😃